Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -1078,7 +1078,7 @@ auto FunctionParser<Context>::parseIndexForLocal(uint32_t& resultIndex) -> Parti
 {
     uint32_t index;
     WASM_PARSER_FAIL_IF(!parseVarUInt32(index), "can't get index for local");
-    WASM_VALIDATOR_FAIL_IF(index >= m_locals.size(), "attempt to use unknown local ", index, " last one is ", m_locals.size());
+    WASM_VALIDATOR_FAIL_IF(index >= m_locals.size(), "attempt to use unknown local ", index, " the number of locals is ", m_locals.size());
```

Comment:
nit: please add a comma at the start of `" the number of locals is "` i.e. `", the number of locals is "`.